### PR TITLE
Fix code style issues

### PR DESCRIPTION
**Pull request by larsgrefer**

### CODE REVIEW

File: src/org/isf/admission/test/Tests.java [hunk 1]
```diff
@@ -354,8 +354,8 @@ public void testIoNewAdmission()
 			
 			Admission admission = testAdmission.setup(ward, patient, admissionType, diseaseIn, diseaseOut1, 
 					diseaseOut2, diseaseOut3, operation, dischargeType, pregTreatmentType, 
-					deliveryType, deliveryResult, true);;
-					result = admissionIoOperation.newAdmission(admission);
+					deliveryType, deliveryResult, true);
+            result = admissionIoOperation.newAdmission(admission);
```
Comment (mwithi) [inline src/org/isf/admission/test/Tests.java]: is the conseguent indentation correct? tabs have been converted into spaces

**Comment by larsgrefer** [comment]
@mwithi These are just some of the default inspections of IntelliJ IDEA

File: src/org/isf/admission/test/Tests.java (hunk 1 above)
Comment (mwithi) [reply src/org/isf/admission/test/Tests.java]: I see, so you should also have noticed a number of warnings we could get rid of.

Anyway, is the above conseguent indentation correct? Tabulations have been converted into spaces maybe

Comment (larsgrefer) [reply src/org/isf/admission/test/Tests.java]: Seems like some tabs were replaced with spaces. I see if I can fix this.

**Comment by larsgrefer** [comment]
@mwithi I've re-created the changes with the correct indentation and added a .editorconfig file, so all editorconfig-aware editors will use tabs from now on.

File: src/org/isf/admission/test/Tests.java (hunk 1 above)
Comment (mwithi) [reply src/org/isf/admission/test/Tests.java]: ok

**Pull request merged**